site/content/en/docs/Integration Patterns/high-density-gameservers.md

```diff
@@ -6,8 +6,7 @@ weight: 70
 description: >
   How to run multiple concurrent game sessions in a single GameServer process.
 ---
-{{< beta title="Allocation State Filter" gate="StateAllocationFilter" >}}
-
+{{% feature publishVersion="1.32.0" %}}
```

Comment:
What we would want to do here instead is use `feature expiryVersion` to wrap the `beta` shortcode - since we want to remove it up until the next release.

The way it is currently would mean that as soon as we pushed this live, the whole block of docs would be removed.